### PR TITLE
Sn readme

### PR DESCRIPTION
**Pull request by NakiNorton**
#### What changes/additions are a part of this PR?
Write README for project
Adds images and gifs that were added in the last merge
### Any background context you want to provide?
This isn't the finished version. I'd like another pair of eyes on it!
#### Where should the reviewer start?
README file
#### How should this be tested (test suite/manually)?
Check that images and gifs are displaying
#### What are the relevant tickets?
n/a
#### Screenshots (if helpful)
n/a
#### Any questions/additional comments:
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,56 @@
...
+2. CD into this repo's directory
+3. Run npm install
+4. Run npm start
+
+**Login to the application for the full user experience:**  
+* username: Diana@turing.io  
+* password: 111111
+
+##  In Action
+#### Homepage:
```
[the remark this answers is not in the document]
Comment (NakiNorton) [reply README.md:32]: Ok I'll see if there's a way to set an image size in the md file, otherwise I'll change the actual image size

**Comment by NakiNorton** [comment]
I modified the size of the images to match the width of the gifs. It looked good in the README preview so hopefully that's fixed now!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (NakiNorton) [reply README.md:32]: I've modified the size of the images to match the gifs. It looked good in the README preview so hopefully that'll be fixed now!

[the remark this answers is not in the document]
Comment (rwilliams659) [reply README.md:32]: Perfection!!

**Pull request merged**